api/config.py

```diff
@@ -84,22 +85,22 @@
 ).get_default_database()
 log.debug(str(db))
 
+es = elasticsearch.Elasticsearch([__config['persistent']['elasticsearch_host']])
```

Comment:
Does this basically hard-code the elasticsearch host, and never pull it from the database?